diff -Nur squashfs1.1b.org/squashfs-tools/mksquashfs.c squashfs1.1b/squashfs-tools/mksquashfs.c
--- squashfs1.1b.org/squashfs-tools/mksquashfs.c	Thu Jan  9 03:07:56 2003
+++ squashfs1.1b/squashfs-tools/mksquashfs.c	Sun Jan 19 20:24:07 2003
@@ -190,7 +190,7 @@
 {
 	off_t off = byte;
 
-	if(lseek64(fd, off, SEEK_SET) == -1) {
+	if(lseek(fd, off, SEEK_SET) == -1) {
 		perror("Lseek on destination failed");
 		exit(0);
 	}
@@ -209,7 +209,7 @@
 	if(off + bytes > ((long long)1<<32) - 1 )
 		BAD_ERROR("Filesystem greater than maximum size 2^32 - 1\n");
 
-	if(lseek64(fd, off, SEEK_SET) == -1) {
+	if(lseek(fd, off, SEEK_SET) == -1) {
 		perror("Lseek on destination failed");
 		exit(0);
 	}
